client/less/challenge.less

```diff
@@ -111,3 +111,67 @@ iframe.iphone {
     right: -207px;
   }
 }
+
+
+.night {
```

Comment:
That is what was in the diff on the original pr. I can adjust it so that it is just one though as well